include/ofi_util.h

```diff
@@ -1008,6 +1008,7 @@ struct ofi_ops_flow_ctrl {
 	size_t	size;
 	void	(*set_threshold)(struct fid_domain *domain, uint64_t threshold);
 	void	(*add_credits)(struct fid_ep *ep, uint64_t credits);
+	ssize_t	(*enable)(struct fid_ep *ep);
```

Comment:
This should return int.  ssize_t is mostly used with data transfer operations (for possible future extensibility).  int is used for other cases.